### PR TITLE
[#167591482] Add translation

### PR DESCRIPTION
**Pull request by valentinacala**
This PR introduces the transaltion of the markdown displayed on the IdentificationScreen

image

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#167591482](https://www.pivotaltracker.com/story/show/167591482): Tradurre il contenuto del markdown pin_unlock_help

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: locales/en/pin_unlock_help.md [hunk 1]
```diff
@@ -1,9 +1,7 @@
-Il PIN è il codice di sicurezza che mantiene nell'app IO i tuoi dati protetti.
+The PIN is the security code that grant your data being securely stored into the IO app.
```
Comment (Undermaken) [inline locales/en/pin_unlock_help.md]:
```suggestion
The PIN is the security code that grants your data is securely stored into the IO app.
```

File: locales/en/pin_unlock_help.md [hunk 2]
```diff
@@ -1,9 +1,7 @@
...
 
-Lo hai scelto quando sei entrato per la prima volta nell'app tramite la tua identità SPID.
+You chose it when you logged in for the first time with your SPID credentials.
 
-Se non lo ricordi, puoi modificarlo in ogni momento  semplicemente premendo il pulsante in basso e usando nuovamente la tua identità SPID.
+If you forgot it, you can change it at any time by pressing the "Did you forget the PIN?" button on the bottom and logging in again by your SPID credentials.
```
Comment (Undermaken) [inline locales/en/pin_unlock_help.md]:
```suggestion
If you forgot it, you can change it at any time by pressing the "Did you forget the PIN?" button on the bottom and logging in again with your SPID credentials.
```

Comment (valentinacala) [reply locales/en/pin_unlock_help.md]: I'm not sure about the usage of "by" with login. Looking on transactions it seems to be more appropriate the usage of "with"

Comment (Undermaken) [reply locales/en/pin_unlock_help.md]: @valentinacala 
> I'm not sure about the usage of "by" with login. Looking on transactions it seems to be more appropriate the usage of "with"

So it looks like better as I suggested? Anyway I guess it's correct in both ways

Comment (valentinacala) [reply locales/en/pin_unlock_help.md]: Yes, it seems to be. committed!

**Pull request merged**